backend/controllers/prisonerProfile/prisonerQuickLook.js

```diff
@@ -132,6 +125,18 @@ module.exports = ({ prisonerProfileService, prisonApi, telemetry }) => async (re
 
   trackEvent(telemetry, username, activeCaseLoad)
 
+  const extractLifeImprisonmentStatus = async () => {
+    const [prisonerDetailsResponse] = await Promise.all(
```

Comment:
I kept this call separate from the other api calls (calls the elstic prisoner search service).. as it is only used to determine if the offender is on a life sentence (at the moment). Therefore only runs in the specific scenario when it's needed.

Although, it does return a lot of the data already on the quicklook tab and is faster than calling the prison-api. So we could refactor to use this call to populate most (maybe all) of the data